cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -2113,7 +2113,12 @@ macro(build_lz4)
 endmacro()
 
 if(ARROW_WITH_LZ4)
-  resolve_dependency(Lz4 PC_PACKAGE_NAMES liblz4)
+  set(ARROW_LZ4_REQUIRED_VERSION "1.8.0")
```

Comment:
Do we need this now that the call to `LZ4F_compressionLevel_max` is guarded by an `#if`?

Reply:
No we do not, I'll revert these changes.